packages/addon-dev/sample-rollup.config.js

```diff
@@ -1,4 +1,4 @@
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
```

Comment:
I think these are the same?

https://github.com/rollup/plugins/blob/5363f55aa1933b6c650832b08d6a54cb9ea64539/packages/babel/src/index.js#L258-L260

Reply:
I forgot about this -- I had made the change because the default export was undefined -- and I "just wanted to focus on the important bits" of what I was working on and not deal with sillyness. But yeah, I forgot to check if this was actually needed when I got the tests passing locally.

Checking now tho (locally):
 - changing the import in v2-addon-dev-test back to use the default import
 -  default import reveals: `TypeError: babel is not a function` -- which is suspicious, because your link points to the same value for both named and default exports....
    - my next hunch is that the embroider-monorepoe's typescript setup is doing dirty things to default imports/exports due to the compile target not being ESM

Reply:
reverted -- unrelated to this work anyway. If we need to update the sample, that should be separate PR